scipy/stats/_stats_py.py

```diff
@@ -9440,3 +9442,88 @@ def rankdata(a, method='average', *, axis=None, nan_policy='propagate'):
         result[nan_indexes] = np.nan
 
     return result
+
+
+# TODO: axis parameter
+# TODO: add _axis_nan_policy_factory decorator
+def expectile(a, alpha=0.5, axis=0, dtype=None, weights=None):
+    r"""Compute the expectile along the specified axis.
+
+    The expectile at level alpha of the array :math:`a_i` associated to
```

Comment:
$\alpha$
On the other hand, others would ask you to remove the LaTeX from here and include it in the notes. (Technically, that's the only place LaTeX is supposed to appear because this is supposed to be readable in the console.) So consider doing that preemptively. Up here, you'd just say that the expectile is a generalization of the mean; see definition in notes.

Reply:
Do you mean unicode alpha or math alpha?
My original intention was to use the written `alpha` to make the 1-1 connection to the α in the formula.

Reply:
I see. In that case: "variables", which we usually take to mean parameters of the present function, go in single backticks. https://numpydoc.readthedocs.io/en/latest/format.html
Here's the part about including equations in the Notes section.
https://numpydoc.readthedocs.io/en/latest/format.html#notes